### ISSUE TITLE
Disabling breadth_first publishing in driver with config file, requires empty string

### ISSUE THREAD

**Issue by kmorri09** [VOLTTRON/volttron#1332]
I wanted to disable the non-historian publishes from the master driver (breadth, breadth_all, depth_all), so I passed into the config file the required entries:

` "publish_breadth_first": "False" (and all variations of the word "false." Passing FALSE without quotes caused a JSON error)`

However, when I tested to make sure the associated variable in the driver was correct, it was set to True. I discovered this was because the driver code uses:

`publish_depth_first = bool(self.config.get("publish_depth_first", True))`

Any string passed into the config file will become True when using bool(). The only way I found to set this to false was to pass in the empty string:

`"publish_breadth_first": ""`

This will become False.

In summary: bool("False") = True, but bool("") = False. I must set all these variables to the empty string in order to limit the excess publishes from the driver. Possible I'm missing something but I could not configure the driver as desired using the config file any other way.

Thanks,
Kevin Morrissey, UW

**Comment by kmonson** [comment]
In JSON you would use all lower case false with no quotes. It's kind of picky that way.

`"publish_breadth_first": false`

https://en.wikipedia.org/wiki/JSON#Data_types.2C_syntax_and_example

We convert the result of the parsed JSON value to  a bool in case someone decides to set it as an integer value.
